pkg/tsdb/elasticsearch/client/client.go

```diff
@@ -168,7 +168,7 @@ func (c *baseClientImpl) executeRequest(method, uriPath, uriQuery string, body [
 	return resp, nil
 }
 
-func (c *baseClientImpl) ExecuteMultisearch(r *MultiSearchRequest) (*MultiSearchResponse, error) {
+func (c *baseClientImpl) ExecuteMultisearch(r *MultiSearchRequest) (*MultiSearchResponse, backend.ErrorSource, error) {
```

Comment:
might be a bit cleaner to have a custom error like [this](https://www.digitalocean.com/community/tutorials/creating-custom-errors-in-go#collecting-detailed-information-in-a-custom-error)